orchagent/muxorch.h

```diff
@@ -196,13 +202,16 @@ class MuxOrch : public Orch2, public Observer, public Subject
 
     void addNexthop(NextHopKey, string = "");
     void removeNexthop(NextHopKey);
+    bool containsNextHop(const NextHopKey&);
+    bool isMuxNexthops(const NextHopGroupKey&);
     string getNexthopMuxName(NextHopKey);
     sai_object_id_t getNextHopId(const NextHopKey&);
 
     sai_object_id_t createNextHopTunnel(std::string tunnelKey, IpAddress& ipAddr);
     bool removeNextHopTunnel(std::string tunnelKey, IpAddress& ipAddr);
     sai_object_id_t getNextHopTunnelId(std::string tunnelKey, IpAddress& ipAddr);
 
+    void updateRoute(const IpPrefix &pfx, bool remove);
```

Comment:
general convention is `bool add=True`, Use 'false' as non-default case during remove